net-mgmt/pfSense-pkg-arpwatch/files/usr/local/pkg/arpwatch.inc

```diff
@@ -224,6 +230,16 @@ fclose($fd);
 if (false !== $message) {
 	$subject = array();
 	preg_match('/^Subject: (.*)$/m', $message, $subject);
+	if (is_array($config['installedpackages']['arpwatch']['config'][0]['row'])) {
```

Comment:
Rather than doing `is_array()`, You should just run that whole bit through `init_config_arr()`. Otherwise if any of the intermediate parts are missing/unset, this will result in a PHP error.

Also the code being generated that this is inside of needs a `global $config;` line.

Reply:
Done, fixed

Reply:
+ added `global $config;`